kustomize/base/kubernetes-manifests/cartservice.yaml

```diff
@@ -40,7 +40,6 @@ spec:
             drop:
               - all
           privileged: false
-          readOnlyRootFilesystem: true
```

Comment:
**Issue:**

Sorry, Jasper.
Let's bring this `readOnlyRootFilesystem: true` back.
I realized _why_ the `cartservice` was causing issues. It's because `readOnlyRootFilesystem: true` doesn't work with the `v0.3.8` `cartservice` image.
So, I'm now creating a new release (`v0.3.9`) of Online Boutique — which will contain the latest source code of `cartservice` (from `main`). See https://github.com/GoogleCloudPlatform/microservices-demo/pull/938.

You will have to:
1. update the images used in your pull-request from `v0.3.8` to `v0.3.9`.
2. wait until https://github.com/GoogleCloudPlatform/microservices-demo/pull/938 is merged before merging your pull-request.

Let me know if I should clarify.

Reply:
Update: https://github.com/GoogleCloudPlatform/microservices-demo/pull/938 is now merged.
Thanks, Olivier, for the speedy review!